cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/BaseStorage.java

```diff
@@ -60,10 +60,6 @@ public String allocateTableLocation(
       String databaseId, String tableId, String tableUUID, String tableCreator) {
     Preconditions.checkArgument(databaseId != null, "Database ID cannot be null");
     Preconditions.checkArgument(tableId != null, "Table ID cannot be null");
-    Preconditions.checkArgument(tableUUID != null, "Table UUID cannot be null");
```

Comment:
this change seem regressive, its valid to ensure tableUUID is not null/ storageproperties is correctly configured

Reply:
Yes, I was a bit confused here. I removed it because it was failing [this](https://github.com/linkedin/openhouse/blob/5ba28ac94bfb064bb00314e75ba67655b431bed4/services/tables/src/test/java/com/linkedin/openhouse/tables/e2e/h2/RepositoryTest.java#L331) test because the `STAGED_TABLE_DTO` did not have a `UUID`; there is probably a better solution here but I wasn't sure what was best